pyproject.toml

```diff
@@ -68,11 +68,12 @@ select = ["ALL"]
 "src/rekor_types/__init__.py" = [
     "TCH001", # False positive: imports are re-exports, not just for type hints.
 ]
-"src/rekor_types/_internal/*.py" = [
+"src/rekor_types/_internal/**/*.py" = [
     "A003",   # some fields shadow python builtins
     "E501",   # handled by black, and catches some docstrings we can't autofix
     "ERA001", # false positives
     "D400",   # overly opinionated docstrings
+    "D205",   # 1 blank line required between summary line and description
```

Comment:
This would be the first time that the genrated docstrings need to be multiline, but so far the formatters can't autofix.